etc/profile

```diff
@@ -61,6 +61,13 @@ MINGW64)
   ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
   MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
   ;;
+ARM64)
+  MINGW_MOUNT_POINT="${MINGW_PREFIX}"
+  PATH="${MINGW_MOUNT_POINT}/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
```

Comment:
I would suggest to add `/mingw32/bin` between the `MINGW_MOUNT_POINT/bin` and the `MSYS2_PATH`: We still might rely on a few MINGW programs that are not available for ARM64 (but for i686, and Windows/ARM64 can emulate that, right?).

Reply:
For sure, @dennisameling proved that even if we leave some i686 components there it's still way faster.